lms/security.py

```diff
@@ -120,19 +146,25 @@ def identity(self, request):
 
         return Identity("", [])
 
-    def permits(self, request, context, permission):
-        return _permits(self, request, context, permission)
+    def permits(self, request, _context, permission):
+        return _permits(self.identity(request), permission)
 
 
-def _permits(policy, request, _context, permission):
-    identity = policy.identity(request)
+def _permits(identity, permission):
     if identity and permission in identity.permissions:
         return Allowed("allowed")
 
     return Denied("denied")
 
 
-def _get_lti_user(request) -> LTIUser:
+def _get_lti_user_from_lti_launch_params(request) -> LTIUser:
```

Comment:
Apologies because I know this name was my suggestion but we can probably get away with avoiding the repetition of `lti` and it'll still be clear enough:

```suggestion
def _get_lti_user_from_launch_params(request) -> LTIUser:
```